ga4gh/datarepo.py

```diff
@@ -442,15 +495,19 @@ def insertReferenceSet(self, referenceSet):
             VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?);
         """
         cursor = self._dbConnection.cursor()
-        cursor.execute(sql, (
-            referenceSet.getId(), referenceSet.getLocalId(),
-            referenceSet.getDescription(), referenceSet.getAssemblyId(),
-            referenceSet.getIsDerived(), referenceSet.getMd5Checksum(),
-            referenceSet.getNcbiTaxonId(),
-            # We store the list of sourceAccessions as a JSON string. Perhaps
-            # this should be another table?
-            json.dumps(referenceSet.getSourceAccessions()),
-            referenceSet.getSourceUri(), referenceSet.getDataUrl()))
+        try:
+            cursor.execute(sql, (
+                referenceSet.getId(), referenceSet.getLocalId(),
+                referenceSet.getDescription(), referenceSet.getAssemblyId(),
+                referenceSet.getIsDerived(), referenceSet.getMd5Checksum(),
+                referenceSet.getNcbiTaxonId(),
+                # We store the list of sourceAccessions as a JSON string.
+                # Perhaps this should be another table?
```

Comment:
Yes, considering we want to support searching on this column, we'll probably want to move it to its own table.